supabase/migrations/20260902132624_expand_webnative_onboarding.sql

```
@@ -0,0 +1,62 @@
+  AND (
+    (NOT ("onboarding" ? 'development_environment'::"text"))
+    OR (
+      ("jsonb_typeof"(("onboarding" -> 'development_environment'::"text")) = 'string'::"text")
+      AND (("onboarding" ->> 'development_environment'::"text") = ANY (ARRAY['hosted_builder'::"text", 'local_project'::"text", 'exploring'::"text"]))
+    )
+  )
+  AND (
+    (NOT ("onboarding" ? 'intent'::"text"))
+    OR (
+      ("jsonb_typeof"(("onboarding" -> 'intent'::"text")) = 'string'::"text")
+      AND (("onboarding" ->> 'intent'::"text") = ANY (ARRAY['ota'::"text", 'builder'::"text", 'both'::"text", 'exploring'::"text", 'publish'::"text"]))
```

SQLFluff (4.3.0)

[error] Line is too long (151 > 80).

(LT05)